### PR TITLE
Add Ruthiel's profile

### PR DESCRIPTION
**Pull request by ruthiel**
- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1528"></a>

### CODE REVIEW

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good, thanks for contributing 👍

File: public/data/ruthiel.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/tofucru",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Reddit",
+        "url": "https://reddit.com/tofucru",
```
Comment (kumarsonsoff3) [inline public/data/ruthiel.json:19]: **There is a broken link. Please update it with the correct one!** 
image


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/ruthiel.json (hunk 1 above)
Comment (ruthiel) [reply public/data/ruthiel.json]: Oh!
Thank you for your review!
Fixed already!

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**